.github/workflows/release.yml

```diff
@@ -1,12 +1,9 @@
 name: Release Latest Version
 
 on:
-  workflow_dispatch:
-    inputs:
-      ref:
-        description: branch, tag or SHA to checkout.
-        required: false
-        default: 'main'
+  push:
+    branches:
+      - main
```

Comment:
I don't think this Action works anymore, as it calls `./ops/cli.py tag-and-push-images` which was removed to moved to `gotc-deploy`, See https://github.com/broadinstitute/gotc-deploy/pull/274. We may need to think about how to release from `gotc-deploy`, or add the cli back.

Reply:
Ugh why did that get absorbed into cli.py????

Reply:
`gotc-deploy` should not control software releases.

Reply:
Thank you - I've restored it.